astroquery/vizier/tests/test_vizier_remote.py

```diff
@@ -94,7 +96,7 @@ def test_query_two_wavelengths(self):
         v = vizier.core.Vizier(
```

Comment:
I only comment on this as this is a refactoring/cleanup PR. But I think, whenever possible we need to switch the tests to use the public API. E.g. import Vizier directly from the namespace and then use it as such, rather than including the implementation details (e.g. the 'core' namespace).

Another thing, that is more an annoyance than anything practical, is to get rid of the single letter, or abreviated variables as we're not in the punchcard era any more, and it would be nicer to grep for things.